src/components/Task/Modal/Form/LogForm/TaskModalLogForm.tsx

```diff
@@ -194,7 +194,8 @@ const TaskModalLogForm = ({
       attributes.result = ResultEnum.Completed;
     }
     // Remove taskPhase from attributes as we don't save it on the DB
-    delete attributes.taskPhase;
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { taskPhase, ...newAttributes } = attributes;
```

Comment:
This should appease ESLint. It ignores this rule for variables that start with an underscore.

```suggestion
    const { taskPhase: _, ...newAttributes } = attributes;
```